src/utils.js

```diff
@@ -408,10 +409,24 @@ function normalizeSourceMapAfterPostcss(map, resourceContext) {
   return newMap;
 }
 
+function isFileExists(filePath, existsSync = fs.existsSync) {
```

Comment:
Do we need default value for any cases?

Reply:
You're right, I removed the default value. They don't seem to be needed here.